_scripts/install.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
the name of this script is install.sh, however it will execute the tool in a temporary location, without moving it to a path that is included in `$PATH` by default or will not even add the tool to the `$PATH` permanently. I would suggest to use run.sh for example, AND/OR provide also a way to have it installed truly.